test/conformance/ptp/ptp.go

```diff
@@ -279,6 +281,32 @@ var _ = Describe("[ptp]", func() {
 					waitUntilLogIsDetected(testPtpPod, 3*time.Minute, "Profile Name: test-slave")
 				})
 			})
+
+			It("Should retrieve the details of hardwares for the Ptp", func() {
+				By("Getting the version of the OCP cluster")
+
+				ocpVersion, err := getOCPVersion()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(ocpVersion).ShouldNot(BeEmpty())
+
+				By("Getting the version of the PTP operator")
+
+				ptpOperatorVersion, err := getPtpOperatorVersion()
+				Expect(err).ToNot(HaveOccurred())
+				Expect(ptpOperatorVersion).ShouldNot(BeEmpty())
```

Comment:
ptpoperator version will be empty if using make deploy, do not fail as we only collect data

Reply:
Did you verify this based on David's comments?